src/libs/antares/study/area/list.cpp

```diff
@@ -326,6 +331,30 @@ bool saveAreaOptimisationIniFile(const Area& area, const Clob& buffer)
     return ini.save(buffer);
 }
 
+bool saveAreaAdequacyPatchIniFile(const Area& area, const Clob& buffer)
+{
+    IniFile ini;
+    IniFile::Section* section = ini.addSection("adequacy-patch");
+    int value;
+    switch (area.adequacyPatchMode)
+    {
+    case Data::AdequacyPatch::adqmVirtualArea:
+        value = 0;
```

Comment:
I can see that when saving **adequacy-patch.ini**, we have something like : 
```ini
[adequacy-patch]
adequacy-patch-mode = 2
```
The value **2** is not very expressive. We could have **inside adq patch domain** for instance